build.sbt

```diff
@@ -42,6 +42,7 @@ libraryDependencies ++= Seq(
   "com.google.oauth-client" % "google-oauth-client" % googleClientApiV,
   "mysql" % "mysql-connector-java" % "5.1.35",
   "org.scalaz" % "scalaz-core_2.11" % "7.1.3",
+  "com.typesafe.play" %% "play-json" % "2.4.3",
```

Comment:
They made a standalone module for their scala json parsing sutff out of the whole framework, which I think is pretty good which why I used it.
But I completely agree since we already have Spray there's not point adding this.